src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -10,6 +10,7 @@
 
 import warnings
 import torch
+from torch.onnx._internal import jit_utils
```

Comment:
I understand that this import provides the functionality required by the decoder but I'm a little worried that you're importing something internal. Can you please double check if there are any remarks in the imported module that it's not recommended to import it or that it's something that might change in the future without a warning?

Reply:
Removed import and changed implementation to use methods provided by torch object.